include/zephyr/cpu_freq/cpu_freq.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2025 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SUBSYS_CPU_FREQ_H_
+#define ZEPHYR_SUBSYS_CPU_FREQ_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <zephyr/types.h>
+#include <zephyr/cpu_freq/pstate.h>
+
```

Comment:
your headers need to be organized in doxygen groups (probably one for CPU Frequency Scaling and one for CPU Load, I'm guessing) that are "mounted" into the [`os_services` group](https://docs.zephyrproject.org/latest/doxygen/html/group__os__services.html)